dist/betteR20-core.user.js

```diff
@@ -2027,6 +2027,7 @@ function baseTool () {
 				<div id="d20plus-quickdelete" title="BetteR20 - Journal Root Cleaner">
```

Comment:
Didn't read the filename above, apologies--please _do not_ commit the `dist/` versions of the scripts; all this does is create unnecessary churn. 
All the comments above still apply, though

Reply:
Apologies, started this feature a long time ago before I figured out how to deal with dist and wasn't sure how to remove it.